conans/model/conanfile_interface.py

```diff
@@ -0,0 +1,11 @@
+
+
+class ConanFileInterface:
```

Comment:
What for?

Reply:
To restrict the visibility that consumers have to their dependencies. Otherwise consumers might be able to call any method in the dependency conanfile or access any attribute.